public/index.php

```diff
@@ -14,8 +17,12 @@
     unset($path);
 }
 
-// Setup autoloading
-require 'init_autoloader.php';
+// Composer autoloading
+include __DIR__ . '/../vendor/autoload.php';
+
+if (! class_exists(Application::class)) {
+    throw new RuntimeException('Unable to load ZF2. Run `php composer.phar install`.');
```

Comment:
This is the first time `composer.phar` term is used. I think we should write something like

> Type vagrant ssh `composer install` if you are using vagrant. Type docker exec composer install if you are using docker. Type composer install or php composer.phar if you are working in local`